WangscapeTest/TestCalculatorLinear.cpp

```diff
@@ -0,0 +1,93 @@
+#include <gtest/gtest.h>
+#include <random>
+#include <numeric>
+#include <algorithm>
+
+#include <tilegen/alpha/CalculatorLinear.h>
+
+class TestCalculatorLinear : public ::testing::Test
+{
+public:
+    tilegen::alpha::CalculatorLinear cl;
+    std::mt19937 rng;
+    TestCalculatorLinear()
+    {
+        std::random_device rd;
+        rng.seed(rd());
+    }
+    ptrdiff_t validateResults(const tilegen::alpha::CalculatorBase::Weights& weights)
+    {
+        cl.updateAlphas(weights);
+        const auto& alphas = cl.getAlphas();
+        int total_alpha = std::accumulate(alphas.cbegin(), alphas.cend(), 0);
+        EXPECT_EQ(255, total_alpha);
+        double total_weight = std::accumulate(weights.cbegin(), weights.cend(), 0.);
+        for (size_t i = 0; i < 4; i++)
+        {
+            double alpha_fraction = double(alphas[i]) / 255;
+            double weight_fraction = double(weights[i]) / total_weight;
+            EXPECT_NEAR(alpha_fraction, weight_fraction, 1.5 / 255);
```

Comment:
Why does this "epsilon" equal `1.5 / 255.0`? Is it an arbitrary value or has it some not so obvious justification?